packages/tomls/src/periphery/configs/tokens-testnet/wbtc.toml

```diff
@@ -0,0 +1,120 @@
+#####################################
```

Comment:
remind me: why did we need additional files for these?

Reply:
these are testnet, they are linked to testnet chains.

Reply:
my question is: why can't we reuse existing toml files for testnet?

Reply:
because they point to mainnet chain IDs

Reply:
mainnet chain ids are configured through vars. The values of vars can differ between mainnet and testnet